params.go

```diff
@@ -100,6 +101,16 @@ func (p *Params) ValidateRequiredProperties() (bool, []error, []string) {
 		errors = append(errors, fmt.Errorf("Timeout %v is not supported; set it between 0 and 540 seconds", p.Memory))
 	}
 
+	supportedIngressSettings := []string{
+	    "",
```

Comment:
Since `all` is the default according to documentation, it would be better to set the value to `all` if it's empty. And then only pass the `--ingress-settings` parameter if it's not equal to `all`.

Reply:
I agree.